main.py

```diff
@@ -827,6 +833,142 @@ def _send_flow_mods(
                 self._send_napp_event(switch, flow, "pending")
         return flow_mods
 
+    def _send_flows_by_switch_from_request(
+        self,
+        request: Request,
+        command: str,
+    ) -> JSONResponse:
+        """Send flows in the respective switches from request."""
+        content_type_json_or_415(request)
+        content_dict = get_json_or_400(request, self.controller.loop)
+        if not isinstance(content_dict, dict):
+            raise HTTPException(400, detail=f"Invalid payload: {content_dict}")
+
+        if not any(content_dict):
+            result = "The request body doesn't have any flows"
+            raise HTTPException(400, detail=result)
+        
+        switches = []
+        for dpid in content_dict:
+            flows = content_dict[dpid]["flows"]
+            if not any(flows):
+                result = f"The switch {dpid} in the request body doesn't have any flows"
+                raise HTTPException(400, detail=result)
+            try:
+                validate_cookies_and_masks(flows, command)
+            except ValueError as exc:
+                raise HTTPException(400, detail=str(exc))
+            
+            switch = self.controller.get_switch_by_dpid(dpid)
+            if not switch:
+                return JSONResponse({"response": "dpid not found."}, status_code=404)
+            switches.append(switch)
+            
+        flows_by_switch_to_log(log.info, command, content_dict)
+        try:
+            errored_switches = self._install_flows_by_switch(
+                switches, content_dict, command
+            )
+            if errored_switches == len(content_dict):
+                msg = "All switches were not connected"
+                raise HTTPException(424, detail=msg)
+            return JSONResponse(
+                {
+                    "response": "FlowMod Message Sent",
+                    "errored_switches": errored_switches
+                },
+                status_code=202)
+        except SwitchNotConnectedError as error:
+            raise HTTPException(424, detail=str(error))
+        except PackException as error:
+            raise HTTPException(400, detail=str(error))
+        except FlowSerializerError as error:
+            raise HTTPException(400, detail=str(error))
+        except ValidationError as error:
+            msg = error_msg(error.errors())
+            raise HTTPException(400, detail=msg) from error
+    
+    def _install_flows_by_switch(
```

Comment:
Here's a challenge for you to solve:

- Can we avoid having to introduce newer duplicated-ish versions of `_install_flows` and `_send_flow_mods` methods? 

Sometimes duplicating it's not too problematic and arguably it's even better than introducing a wrong abstraction, that said, can we even avoid duplication here?

1 - What if we were to change `_install_flows` `flows_dict: dict`  to be a `list[dict]` and then zip with the list of switches? Where each dict would be the flow payload. This would require some call sites being refactored, but shouldn't bee too bad. The biggest drawback would be when sending to all switches it'd create more flows in memory, but eventually we can also further limit how many flows per switch if that ever becomes a problem. 

2 - Let's try to reuse `_send_flow_mods`, what if we only support force being the same for all switches? that would allow us to still reuse most of it and rely on the existing exceptions. In the future we can reassess later if force has a plausible use case for individual switches, one typically ones either all force True or no force at all, a mix is very rare as far as typical use cases, and for the case you're using with mef_eline it'll be force False for installation.